src/schema/v2/recordArtworkView.ts

```diff
@@ -0,0 +1,43 @@
+import { GraphQLNonNull, GraphQLString } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import { GraphQLError } from "graphql"
+
+export interface RecordArtworkViewMutationInput {
+  artwork_id: string
+}
+
+export const recordArtworkViewMutation = mutationWithClientMutationId<
+  RecordArtworkViewMutationInput,
+  any,
+  ResolverContext
+>({
+  name: "RecordArtworkViewMutation",
```

Comment:
Wondering, if you omit `Mutation` here - will it produce the schema without changes that break relay (`RecordArtworkViewInput` -> `RecordArtworkViewMutationInput`)

Reply:
Hmmm, now that I look closer it seems like we're a bit inconsistent with suffixing these names with `Mutation` in the file itself... I wonder if relay adds Mutation anyway but I'll check.